frontend/packages/operator-lifecycle-manager/src/components/console-operator-config.tsx

```diff
@@ -0,0 +1,214 @@
+import * as React from 'react';
```

Comment:
It looks like we have this in the PR twice now?

Reply:
Good catch! I orphaned this when I was moving files around.  This copy is now deleted.